pages/admin.js

```diff
@@ -1,26 +1,71 @@
-import ArtPieceForm from "@/components/ArtPieceForm/ArtPieceForm";
+import ArtPieceForm from "@/components/AdminArtPieceForm/ArtPieceForm";
 import Header from "@/components/Header/Header";
 import ArtPiecesList from "@/components/AdminArtPiecesList/AdminArtPiecesList";
 
-export default function AdminHomePage({ artPieces, setArtPieces }) {
-  function addArtPiece(newArtPieceData) {
+export default function AdminHomePage({
+  artPieces,
+  artPieceToEdit,
+  handleSetArtPieces,
+  handleArtPieceToEdit,
+  fileImageUrl,
+  handleSetFileImageUrl,
+}) {
+  const allowedFileSize = 600000;
```

Comment:
Let's make it a round 600 KB. Since everything is in binary, you have to multiply 1024 bytes (= 1 KB) by some number, like 600, and you get that many bytes back.
```suggestion
  const allowedFileSize = 614400;
```